types/django-bananas/index.d.ts

```diff
@@ -0,0 +1,222 @@
+}
+
+export function useAdmin(context: AdminContext): AdminInterface;
+export type AdminContext = React.Context<AdminInterface>;
+
+interface ContainerProps {
+    className?: string;
+}
+export class Container extends React.Component<ContainerProps> {}
+
+interface ContentProps {
+    disablePadding?: boolean;
+    contained?: boolean;
+    className?: string;
+}
+export class Content extends React.Component<ContentProps> {}
+
+interface LinkProps {
+    children: string | React.ReactNode;
+    route?: string;
+    params?: Record<string, string | number>;
+    path?: string;
+    query?: string | Record<string, string | number>;
+    hash?: string;
+    href?: string;
+    patch?: boolean;
+    passHref?: boolean;
+}
+export class Link extends React.Component<LinkProps> {}
+
+interface PageInterface<T> {
+    title: string;
+    route: RouteData;
+    referer?: string;
+    data: PageData<T>;
+}
+
+interface PageData<T> {
+    obj: T;
+    body: T;
+    headers: Record<string, string>;
+    status: number;
+    statusText: string;
+    ok: boolean;
+    url: string;
+    text: string;
+}
+
+export function usePage<T>(context: PageContext<T>): PageContext<T>;
+type PageContext<T> = React.Context<PageInterface<T>>;
+
+interface LoginFormProps {
+    logger: any;
+}
+export class LoginForm extends React.Component<LoginFormProps> {}
+
+interface TitleBarProps {
+    overrides?: any;
+    children?: React.ReactNode[] | React.ReactNode;
+    color?: 'primary' | 'secondary' | 'paper';
+    title?: string;
+    back?: boolean | string;
+    dense?: boolean;
+    justify?: 'start' | 'center' | 'end' | 'between' | 'around' | 'evenly';
+}
+export const TitleBar: React.FC<TitleBarProps>;
+
+interface ToolBarProps {
+    className?: string;
+}
+export class ToolBar extends React.Component<ToolBarProps> {}
+
+interface ToolsProps {
+    children?: React.ReactNode | React.ReactNode[];
+    className?: string;
+}
+export class Tools extends React.Component<ToolsProps> {}
+
+export function t(s: string, params: any): string;
+
+interface TranslateProps {
+    children: string;
+    params: any;
+}
+export const Translate: React.FC<TranslateProps>;
+
+interface RouteData {
+    hash?: string;
+    id?: string;
+    params?: Record<string, string | number>;
+    path?: string;
+    query?: Record<string, string>;
+}
+
+interface PageInterfaceBase {
+    title: string;
+    route: RouteData;
+    referer?: string;
+}
+
+interface ListPageInterface<T> extends PageInterfaceBase {
+    data: PageData<T[]>;
+}
+
+interface PageInterface<T> extends PageInterfaceBase {
+    data: PageData<T>;
+}
+
+interface NavItem {
+    title?: string;
+    icon?: React.ReactNode;
+}
+
+interface NavAppItem {
+    showSubheader?: boolean;
+}
+
+type ApiSetting =
+    | string
+    | {
+          url: string;
+          requestInterceptor?: (v: Request) => Request;
+          responseInterceptor?: (v: Response) => Response;
+      };
+
+interface AlertProps {
+    classes: object;
```

Comment:
same as with `Theme`, it's not any object, it's a keyed interface with props, no methods.